test/logs_aggregator/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+services:
+  loki:
```

Comment:
add `container_name` for each service else the container default name can be quite long ie :
-  `  logs_aggregator-loki-1` 
-  `  logs_aggregator-grafana-1` 
-  `  logs_aggregator-promtail-1`

Reply:
gotcha,it will be now:
 ✔ Container localnet-loki      Started                                                                                                       0.6s 
 ✔ Container localnet-grafana   Started                                                                                                       0.7s 
 ✔ Container localnet-promtail  Started